### PR TITLE
Accessibility features for markdown files

### PR DESCRIPTION
**Pull request by mishmanners**
Add accessibility features for all markdown based on: https://github.blog/2023-10-26-5-tips-for-making-your-github-profile-page-accessible/

Checked all markdown files in the main folder, others are all okay :wink:

### CODE REVIEW

**Review by diehuxx — APPROVED** [review]
thanks

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -118,19 +118,10 @@ community.
...
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
-version 2.1, available at
-[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org),
+version 2.1, available on [the contributor Convenant website](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html).
 
 Community Impact Guidelines were inspired by
-[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
+[Mozilla's code of conduct enforcement ladder](https://github.com/mozilla/diversity).
```
Comment (EbonyLouis) [inline CODE_OF_CONDUCT.md:125]: switch this url to: https://github.com/mozilla/inclusion/blob/master/code-of-conduct-enforcement/consequence-ladder.md

Comment (frankhinek) [reply CODE_OF_CONDUCT.md:125]: Done

**Comment by frankhinek** [comment]
@mishmanners I attempted to make these modifications but it appears you have the [Allowing changes to a pull request branch created from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) disabled.

**Comment by mishmanners** [comment]
I'll do that today!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mishmanners) [reply CODE_OF_CONDUCT.md:125]: will do.

**Comment by mishmanners** [comment]
Should be fixed now @frankhinek

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -118,19 +118,10 @@ community.
...
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
-version 2.1, available at
-[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org),
+version 2.1, available on [the contributor Convenant website](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html).
 
 Community Impact Guidelines were inspired by
-[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
+[Mozilla's code of conduct enforcement ladder]([https://github.com/mozilla/diversity](https://github.com/mozilla/inclusion/blob/master/code-of-conduct-enforcement/consequence-ladder.md)).
```
Comment (frankhinek) [inline CODE_OF_CONDUCT.md:125]: Duplicate URLs.  Suggested change below you can accept and push through the GitHub interface:

```suggestion
[Mozilla's code of conduct enforcement ladder](https://github.com/mozilla/inclusion/blob/master/code-of-conduct-enforcement/consequence-ladder.md).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ to your valuable work:
   rebase atop the upstream `main` branch. This will limit the potential for merge
   conflicts during review, and helps keep the audit trail clean. A good writeup for
   how this is done is
-  [here](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec), and if you're
+  [written up in our beginners' guide to squashing commits tutorial](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec), and if you're
```
Comment (frankhinek) [inline CONTRIBUTING.md:97]: It isn't a guide that TBD created so removed the "our" portion:
```suggestion
 [this beginner's guide to squashing commits](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec)
```

**Review by frankhinek — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
-# 🎉 **We're participating in Hacktoberfest 2023!** 🎉
+# :tada: **We're participating in Hacktoberfest 2023!** :tada:
 
 Want to contribute during Hacktoberfest? We'd love to have you! Dive in, and your contributions could earn you some exclusive rewards.
```
Comment (frankhinek) [inline README.md:4]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
 
-The **first 20 contributors** to successfully merge a PR will secure exclusive swag of their choosing from our [TBD shop](https://www.tbd.shop/) — we're in the midst of uploading new swag!  Keep an eye on our [leaderboard issue](https://github.com/TBD54566975/developer.tbd.website/issues/721) to see where you rank! ⭐️
+The **first 20 contributors** to successfully merge a PR will secure exclusive swag of their choosing from our [TBD shop](https://www.tbd.shop/) — we're in the midst of uploading new swag!  Keep an eye on our [leaderboard issue](https://github.com/TBD54566975/developer.tbd.website/issues/721) to see where you rank! :star:
```
Comment (frankhinek) [inline README.md:5]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
+🚀 **Gear up for a month packed with exciting events!** :tada:
 
-- Mark your calendars for our **Hacktoberfest Launch event on [October 2nd](https://discord.com/events/937858703112155166/1154126364484583465)**.
+- Mark your calendars for our **Hacktoberfest Launch event on [October 2nd on Discord](https://discord.com/events/937858703112155166/1154126364484583465)**.
 - Stay in the loop - keep an eye on our Discord calendar and pop into our [events-and-updates channel](https://discord.com/channels/937858703112155166/1151972299814215701) regularly! You won't want to miss out!
 
 ## **Hacktoberfest Guidelines:**
 
 - Ensure your contribution is meaningful and fits within the scope of our project, by reading an open issue in its entirety before diving in.
 - Check out our `good-first-issue` and `hacktoberfest` labels in the issues section.
```
Comment (frankhinek) [inline README.md:15]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,19 +1,19 @@
-# 🎉 **We're participating in Hacktoberfest 2023!** 🎉
+# :tada: **We're participating in Hacktoberfest 2023!** :tada:
 
 Want to contribute during Hacktoberfest? We'd love to have you! Dive in, and your contributions could earn you some exclusive rewards.
 
-The **first 20 contributors** to successfully merge a PR will secure exclusive swag of their choosing from our [TBD shop](https://www.tbd.shop/) — we're in the midst of uploading new swag!  Keep an eye on our [leaderboard issue](https://github.com/TBD54566975/developer.tbd.website/issues/721) to see where you rank! ⭐️
+The **first 20 contributors** to successfully merge a PR will secure exclusive swag of their choosing from our [TBD shop](https://www.tbd.shop/) — we're in the midst of uploading new swag!  Keep an eye on our [leaderboard issue](https://github.com/TBD54566975/developer.tbd.website/issues/721) to see where you rank! :star:
 
-🚀 **Gear up for a month packed with exciting events!** 🎉
+🚀 **Gear up for a month packed with exciting events!** :tada:
```
Comment (frankhinek) [inline README.md:8]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
+- Mark your calendars for our **Hacktoberfest Launch event on [October 2nd on Discord](https://discord.com/events/937858703112155166/1154126364484583465)**.
 - Stay in the loop - keep an eye on our Discord calendar and pop into our [events-and-updates channel](https://discord.com/channels/937858703112155166/1151972299814215701) regularly! You won't want to miss out!
 
 ## **Hacktoberfest Guidelines:**
 
 - Ensure your contribution is meaningful and fits within the scope of our project, by reading an open issue in its entirety before diving in.
 - Check out our `good-first-issue` and `hacktoberfest` labels in the issues section.
-- Join our Discord: Connect with the community, stay up to date with Hacktoberfest events/prizes, and discuss Hacktoberfest contributions on our Discord server. Click [here](https://discord.com/channels/937858703112155166/1151216855957123104) to join.
+- Join our Discord: Connect with the community, stay up to date with Hacktoberfest events/prizes, and discuss Hacktoberfest contributions on our Discord server. Click [to join our Discord server](https://discord.com/channels/937858703112155166/1151216855957123104).
```
Comment (frankhinek) [inline README.md:16]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,19 +1,19 @@
...
 - Always be respectful and follow our [code of conduct](https://developer.tbd.website/open-source/code-of-conduct).
 - If in doubt about what to contribute, reach out to maintainers by raising a question in the relevant issue or specified discord channel.
 - **Other participating TBD Repos:**
```
Comment (frankhinek) [inline README.md:19]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -25,9 +25,9 @@ The **first 20 contributors** to successfully merge a PR will secure exclusive s
 
 Celebrate the 10th anniversary of Hacktoberfest this year! Hosted annually every October, Hacktoberfest is a month-long event sponsored by DigitalOcean, GitHub, and various other partners, championing open-source contributions.
```
Comment (frankhinek) [inline README.md:27]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -25,9 +25,9 @@ The **first 20 contributors** to successfully merge a PR will secure exclusive s
 
 Celebrate the 10th anniversary of Hacktoberfest this year! Hosted annually every October, Hacktoberfest is a month-long event sponsored by DigitalOcean, GitHub, and various other partners, championing open-source contributions. 
 
-> ⭐️ If you're new to Hacktoberfest, you can learn more and register to participate [here](https://hacktoberfest.com/participation/). Registration is from **September 26th- October 31st**.
+> :star: If you're new to Hacktoberfest, you can learn more and register to participate [on the Hacktoberfest website](https://hacktoberfest.com/participation/). Registration is from **September 26th- October 31st**.
```
Comment (frankhinek) [inline README.md:29]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -25,9 +25,9 @@ The **first 20 contributors** to successfully merge a PR will secure exclusive s
...
 
-## **New Contributor? Welcome!** 🌟
+## **New Contributor? Welcome!** :star2:
 
 We wholeheartedly embrace new contributors to our community. Remember, every expert was once a beginner, and we understand the initial hurdles you might feel. Here’s how you can dive in:
```
Comment (frankhinek) [inline README.md:33]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -40,9 +40,9 @@ We wholeheartedly embrace new contributors to our community. Remember, every exp
   - Mark your calendars for our **Hacktoberfest Launch event on [October 2nd](https://discord.com/events/937858703112155166/1154126364484583465)**.
   - Stay in the loop - keep an eye on our Discord calendar and pop into our [#events-and-updates channel](https://discord.com/channels/937858703112155166/1151972299814215701) regularly! You won't want to miss out!
 
-Your contribution, be it big or minuscule, carries immense value. We eagerly await to see the waves you'll make in our community! 🚀
+Your contribution, be it big or minuscule, carries immense value. We eagerly await to see the waves you'll make in our community! :rocket:
 
-Here's to a thrilling Hacktoberfest voyage with us! 🎉
+Here's to a thrilling Hacktoberfest voyage with us! :tada:
```
Comment (frankhinek) [inline README.md:46]:
```suggestion
```

File: README.md [hunk 14]
```diff
@@ -40,9 +40,9 @@ We wholeheartedly embrace new contributors to our community. Remember, every exp
   - Mark your calendars for our **Hacktoberfest Launch event on [October 2nd](https://discord.com/events/937858703112155166/1154126364484583465)**.
   - Stay in the loop - keep an eye on our Discord calendar and pop into our [#events-and-updates channel](https://discord.com/channels/937858703112155166/1151972299814215701) regularly! You won't want to miss out!
 
-Your contribution, be it big or minuscule, carries immense value. We eagerly await to see the waves you'll make in our community! 🚀
+Your contribution, be it big or minuscule, carries immense value. We eagerly await to see the waves you'll make in our community! :rocket:
```
Comment (frankhinek) [inline README.md:44]:
```suggestion
```

File: README.md [hunk 15]
```diff
@@ -40,9 +40,9 @@ We wholeheartedly embrace new contributors to our community. Remember, every exp
   - Mark your calendars for our **Hacktoberfest Launch event on [October 2nd](https://discord.com/events/937858703112155166/1154126364484583465)**.
   - Stay in the loop - keep an eye on our Discord calendar and pop into our [#events-and-updates channel](https://discord.com/channels/937858703112155166/1151972299814215701) regularly! You won't want to miss out!
```
Comment (frankhinek) [inline README.md:42]:
```suggestion
```

**Review by frankhinek — CHANGES_REQUESTED** [review]

File: README.md [hunk 16]
```diff
@@ -1,49 +1,15 @@
-# 🎉 **We're participating in Hacktoberfest 2023!** 🎉
```
Comment (frankhinek) [inline README.md:1]:
```suggestion
```

File: README.md [hunk 17]
```diff
@@ -1,49 +1,15 @@
...
-- Check out our `good-first-issue` and `hacktoberfest` labels in the issues section.
-- Join our Discord: Connect with the community, stay up to date with Hacktoberfest events/prizes, and discuss Hacktoberfest contributions on our Discord server. Click [here](https://discord.com/channels/937858703112155166/1151216855957123104) to join.
-- Always be respectful and follow our [code of conduct](https://developer.tbd.website/open-source/code-of-conduct).
-- If in doubt about what to contribute, reach out to maintainers by raising a question in the relevant issue or specified discord channel.
-- **Other participating TBD Repos:**
   - [DWN-Server](https://github.com/TBD54566975/dwn-server/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
   - [DWN-SDK-JS](https://github.com/TBD54566975/dwn-sdk-js/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
   - [developer.tbd.website](https://github.com/TBD54566975/developer.tbd.website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
 
 ## **What is Hacktoberfest?**
```
Comment (frankhinek) [inline README.md:6]:
```suggestion
```

File: README.md [hunk 18]
```diff
@@ -1,49 +1,15 @@
...
-
 - **Join Our Discord Channel**:
   - Once inside, check out the [`Hacktoberfest`](https://discord.com/channels/937858703112155166/1151216855957123104) section. This has all you need: resources, guidelines, and a checklist to help you make your first hacktoberfest contribution.
 - **Feeling Anxious or Unsure? Find a Buddy!**:
   - Head over to our [`hack-together`](https://discord.com/channels/937858703112155166/1151519449837482044) section on Discord. It's perfectly normal to feel a tad overwhelmed or even the impostor syndrome on your first go. In this space, you can partner with someone to collaborate, share thoughts, or jointly tackle an issue. You know what they say, two heads are better than one!
 - **Dive In**:
   - Skim through our [open issues](https://github.com/TBD54566975/developer.tbd.website/edit/main/README.md#hacktoberfest-guidelines) and pick one you vibe with. And if you're on the fence about anything, don't hesitate to ask. Your new community is here to assist and walk with you every step of the way.
```
Comment (frankhinek) [inline README.md:12]:
```suggestion
```

**Review by frankhinek — CHANGES_REQUESTED** [review]

**Review by frankhinek — APPROVED** [review]

**Comment by frankhinek** [comment]
@mishmanners I tried to resolve the merge conflicts in this PR, but it appears you'll have to rebase on the current main branch and push up a commit before this PR can be merged.  The primary reason is that the "Hacktoberfest" header was already removed from `README.md` and this PR still contains references to that text.

Once you rebase, we should be able to merge.  Thanks again for the contribution.

**Comment by mishmanners** [comment]
So this says merged now. All good @frankhinek? Sorry for the delay I was away for a couple of days without much internet.

**Pull request merged**